SUI/SUI/ContentView.swift

```diff
@@ -1,18 +1,156 @@
 //  ContentView.swift
 //  SUI
 //  Created by angelina on 17.01.2023.
-
+import AVFoundation
 import SwiftUI
 
+/// Экран плейера
 struct ContentView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let songName = "gori"
+        static let songTypeString = "mp3"
+        static let shareButtonText = "Поделиться"
+        static let shareButtonMessageText = "Поделиться песней?"
+        static let yesText = "Да"
+        static let saveButtonTitle = "Скачать"
+        static let actionSheetTitle = "Скачивание"
+        static let actionSheetMessageString = "сохранен в папку Загрузки"
+        static let formatString = "%.2f"
+        static let playButtonTitle = "Play"
+        static let stopButtonTitle = "Stop"
+        static let previousImageName = "backward"
+        static let nextImageName = "forward"
+        static let imageCornerRadius: CGFloat = 20
+        static let imageWidth: CGFloat = 350
+        static let imageHeight: CGFloat = 400
+        static let minutesNumber: Double = 60
+        static let butonWidth: CGFloat = 100
+        static let butonHeight: CGFloat = 50
+        static let butonCornerRadius: CGFloat = 50
+    }
+    
+    // MARK: - Public Properties
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            HStack {
+                Spacer()
+                shareButton
+                saveButton
+            }
+            
+            Image(viewModel.setupPhotoName())
+                .resizable()
+                .cornerRadius(Constants.imageCornerRadius)
+                .frame(width: Constants.imageWidth, height: Constants.imageHeight)
+            timeSlider
+            
+            HStack {
+                Text(verbatim: viewModel.getTime().formatted(.dateTime.minute().second())).padding()
+                Spacer()
+                Text(verbatim: Date(timeIntervalSince1970: viewModel.maxDuration - viewModel.currentDuration).formatted(.dateTime.minute().second())).padding()
+            }
+            
+            HStack {
+                startButton
+                stopButton
+            }
+            
+            HStack {
+                previousSongButton
+                nextSongButton
+            }
+        }
+    }
+    
+    // MARK: - Private property
+    
+    @StateObject private var viewModel = PlayerViewModel()
```

Comment:
```suggestion
    @StateObject private var viewModel = PlayerViewModel()
    
```
Разделяем отступами разные враперы между собой